mappings/REMIND-EU_2.0.yml

```diff
@@ -0,0 +1,124 @@
+model:
+  - REMIND-EU 2.0
+native_regions:
+  - CAZ: REMIND 2.0|Canada, Australia, New Zealand
+  - CHA: REMIND 2.0|China and Taiwan
+  - IND: REMIND 2.0|India
+  - JPN: REMIND 2.0|Japan
+  - OAS: REMIND 2.0|Other Asia
+  - LAM: REMIND 2.0|Latin America and the Caribbean
+  - MEA: REMIND 2.0|Middle East and North Africa
+  - REF: REMIND 2.0|Russia and Reforming Economies
+  - SSA: REMIND 2.0|Sub-Saharan Africa
+  - USA: REMIND 2.0|United States of America
+  # 12-region version of REMIND
+  - EUR: REMIND 2.0|EU 28
+  - NEU: REMIND 2.0|Non-EU28 Europe
+  # 21-region version of REMIND
+  - DEU: REMIND 2.0|Germany
+  - FRA: REMIND 2.0|France
+  - UKI: REMIND 2.0|United Kingdom and Ireland
+  - ECE: REMIND 2.0|EU Center-East Europe
+  - ECS: REMIND 2.0|EU Center-South Europe
+  - ENC: REMIND 2.0|EU North-Center Europe
+  - ESC: REMIND 2.0|EU South-Center Europe
+  - ESW: REMIND 2.0|EU South-West Europe
+  - EWN: REMIND 2.0|EU North-West Europe
+  - NEN: REMIND 2.0|Non-EU Northern Europe
+  - NES: REMIND 2.0|Non-EU Southern Europe
+common_regions:
```

Comment:
```
  # exogenously aggregated regions
  - EU27: REMIND 2.0|EU 27
```

Reply:
For clarification: is this really always EU27, or is this some own aggregation based on the native model regions as part of your post-processing?

Reply:
This is a approximation right now (EU27=EUR-UKI), so right now it is EU27 - Ireland actually. However, we intend to improve this in the near term to be able to correctly report EU27.
As in 21 regions we always report the EU27 aggregation anyway, I would say it will be easier to you to simply add this additional native region to avoid the need to revisit that in the future.

Reply:
To avoid confusion during the analysis for the EU Climate Advisory Board whether the scenario data is really EU-27, I'd prefer to not use EU-27 until you can confirm that this is indeed all 27 EU countries (not an approximation).

Reply:
We assume it to be EU-27 in all our project analysis, so please include it as such in our region mapping so we can provide EU27 climate neutral mitigation pathways, which is the aim for this call.

Reply:
FE trajectories, climate targets, and so on for the EU-27 countries are all included in this region results.

Reply:
Ok, thanks for the clarification